misk-aws-dynamodb-testing/build.gradle

```diff
@@ -0,0 +1,21 @@
+dependencies {
+  compile dep.awsApi
+  compile dep.awsSqs
+  compile dep.tracingJaeger
+  compile project(':misk')
```

Comment:
Could you replace `compile` with `implementation` and `testCompile` with `testImplementation`?

Reply:
Is that what we should use going forward? Again, this is what `misk-aws` does

Reply:
Yes: https://stackoverflow.com/questions/44493378/whats-the-difference-between-implementation-and-compile-in-gradle

Reply:
👍